stsx

```diff
@@ -128,13 +130,18 @@ if ! [ "$watchdog_port" -ge 1 -a "$watchdog_port" -le 65535 ]; then
   error "Port number of the watchdog is out of range."
 fi
 
-if [ "x$watchdog" != x ] ; then
+if [ "x$watchdog" != x ]; then
   nc -w 5 -z $watchdog $watchdog_port > /dev/null 2>&1
   if [ $? -ne 0 ]; then
     error "The watchdog is not reachable."
   fi
 fi
 
+user_check=$(cat /etc/passwd | cut -d: -f1 | grep $user)
```

Comment:
Won't work for NIS users. ypcat passwd | ... could be used as plan B. But what about LDAP? No idea! Maybe comment out this check for now with an appropriate comment.

Reply:
Okay, I will think about it and comment out the check for now.